### ISSUE TITLE
STARTTRANSFER_TIME is always 0 when using POST requests with a body

### ISSUE THREAD

**Issue by tnolet** [JCMais/node-libcurl#183]
It seems that the STARTTRANSFER_TIME is always 0 when doing POST requests. With GET everything is fine. I need to use this data to analyse and visualise the cURL request in my application.

You can reproduce using the following code snippets:

This piece of JS code will poop out the 0 value.
```js
const { Curl } = require('node-libcurl');

function doRequest () {
  return new Promise((resolve, reject) => {
    const curl = new Curl();

    curl.setOpt('URL', 'http://requestbin.fullcontact.com/zp1q3azp');
    curl.setOpt(Curl.option.POST, 1);
    curl.setOpt(Curl.option.POSTFIELDS, 'data and more data and more data')
    curl.setOpt(Curl.option.VERBOSE, true);

    curl.on('end', function (statusCode, data, headers) {
      console.info('start transfer time', curl.getInfo('STARTTRANSFER_TIME'));
      this.close();
      resolve(statusCode)
    });

    curl.on('error', function() {
      reject()
      this.close()
    });
    curl.perform();
  })
}

doRequest()
  .then(() => {
    console.log('done')
  })

```

This bash command will show a correctly reported start transfer time using the same request as the JS code above.

```bash
curl -d "data data data" -X POST "http://requestbin.fullcontact.com/zp1q3azp" -vv --write-out 'lookup:        %{time_namelookup}\nconnect:       %{time_connect}\nappconnect:    %{time_appconnect}\npretransfer:   %{time_pretransfer}\nredirect:      %{time_redirect}\nstarttransfer: %{time_starttransfer}\ntotal:         %{time_total}\n' 
```

Running this on MacOS

**Comment by JCMais** [comment]
Thanks for the bringing this to my attention, what was the version of the curl binary you used?

I have just tried the command you sent on multiple libcurl versions, and it seems to (wrongly) return `0` for all versions until `7.64.0`, which is the latest libcurl version that returns a value different than 0.

I have reported it to the libcurl repo, in case you want to follow it's here: https://github.com/curl/curl/issues/4136

**Comment by tnolet** [comment]
@JCMais I was certainly on a version lower than the one you mentioned! So this is a bug I'm afraid.

**Comment by JCMais** [comment]
Fixed upstream, will probably be available only on libcurl `7.56`, I will keep this opened until it releases and I update the release script to use that version for the prebuilt addon.

**Comment by JCMais** [comment]
Newer libcurl version has been released
